app/scripts/controllers/secrets.js

```diff
@@ -19,10 +23,26 @@ angular.module('openshiftConsole')
         $scope.project = project;
         $scope.context = context;
         watches.push(DataService.watch("secrets", context, function(secrets) {
-          $scope.secrets = _.sortBy(secrets.by("metadata.name"), ["type", "metadata.name"]);
```

Comment:
It looks like we've lost the sort?

Reply:
That should be done by the `$scope.unfilteredSecrets = secrets.by("metadata.name");` on the line 27. No ?

Reply:
I don't see the call to `_.sortBy` there?

Reply:
@spadgett updated :)